js/src/exports-common.ts

```diff
@@ -1,13 +1,134 @@
 // Common exports shared between browser and node builds
 export { LazyValue } from "./util";
-export * from "./logger";
-export * from "./functions/invoke";
-export * from "./functions/stream";
+
+export type {
```

Comment:
TIL that `export *` also exported the file's types so we need to specify both the types and objects now separately